README.md

```diff
@@ -110,6 +110,15 @@ Cromshell is a CLI for submitting workflows to a Cromwell server and monitoring/
      * Requires the `bq_cost_table` key to exist in the cromshell 
        configuration file and have a value equal to the BigQuery cost table 
        for your GCP billing project.
+       * For example, using the Methods cromwell server, your `~/.cromshell/cromshell_config.json` 
+         should be as follows:
+         ```json
+         {
+           "cromwell_server": "https://cromwell-v77.dsde-methods.broadinstitute.org/",
+           "requests_timeout": 5,
+           "bq_cost_table": "Methods_billing_dump.gcp_billing_export_v1_009C7D_923007_219A6F"
```

Comment:
I'd prefer we hide these values from the open internet, just in case...

Reply:
so how about the following:
`"<bq_dataset_hosting_the_billing_datatable>.<the_billing_datatable>"`

Reply:
We could do that if we need to yeah

Reply:
(But if there's no risk in exposing these, I'd like the example to be 100% explicit.  Because I never would have foudn those values.
  So... is there a risk?)

Reply:
Or... better... if we included directions for _how_ to find that table.

Reply:
But I don't know what those directions are